src/tools/interopserver/InteropServer.h

```diff
@@ -154,6 +154,7 @@ struct HttpConnection {
         _In_opt_ void* Context,
         _Inout_ QUIC_CONNECTION_EVENT* Event
         ) {
+        UNREFERENCED_PARAMETER(Connection);
```

Comment:
Instead of `UNREFERENCED_PARAMETER` for function inputs that aren't used, just comment out the name of the variable: `_In_ HQUIC /* Connection */`. Same for the other one.